wagn/lib/wagn/railtie.rb

```diff
@@ -0,0 +1,12 @@
+
+require 'decko/engine'
```

Comment:
Engines depend on Railties, so they require railties.  

I'm not up for merging this railtie stuff in without seeing more rationale for this stuff.  I really think it's time to create an engine in a dummy rails app.  No doubt we'll learn a bunch.  Perhaps it will probe necessary to tweak railties, but at present these changes aren't even used (here or in the generator branch), so I can't really justify merging them in.

In the meantime, 1.15 is really being held up by performance and CSS issues, so there's really no rush, now that we're generally clear on namespaces and gem organization.